doozer/doozerlib/repodata.py

```diff
@@ -78,7 +79,7 @@ def from_dict(nvrea_dict: Dict):
         )
 
     @staticmethod
-    def from_metadata(metadata: ET.Element):
+    def from_metadata(metadata: xml.etree.ElementTree.Element):
```

Comment:
What does `defusedxml.ElementTree.fromString` return? We should update the type hint accordingly, or remove it at all, so that references to `xml.etree.ElementTree` can go away.

Reply:
According to the project [documentation](https://pypi.org/project/defusedxml/), it should still return the original `xml.etree.ElementTree.Element`.
> All functions and parser classes accept three additional keyword arguments. They return either the same objects as the original functions or compatible subclasses.

And defusedxml's [source code](https://github.com/tiran/defusedxml/blob/main/defusedxml/ElementTree.py#L151) seems to support it.